planning/behavior_path_dynamic_avoidance_module/include/behavior_path_dynamic_avoidance_module/scene.hpp

```diff
@@ -54,19 +55,34 @@ std::vector<T> getAllKeys(const std::unordered_map<T, S> & map)
 namespace behavior_path_planner
 {
 using autoware_auto_perception_msgs::msg::PredictedPath;
+using autoware_auto_planning_msgs::msg::PathWithLaneId;
 using tier4_autoware_utils::Polygon2d;
 
 struct MinMaxValue
 {
   double min_value{0.0};
   double max_value{0.0};
+  MinMaxValue operator+(const double & scalar) const
+  {
+    MinMaxValue ret;
+    ret.min_value = min_value + scalar;
+    ret.max_value = max_value + scalar;
+    return ret;
+  };
+  void swap() { std::swap(min_value, max_value); };
 };
 
 enum class PolygonGenerationMethod {
   EGO_PATH_BASE = 0,
   OBJECT_PATH_BASE,
 };
 
+enum class ObjectBehaviorType {
+  IGNORE = 0,
+  REGULATED,
+  PRIORITIZED,
+};
```

Comment:
The subject of IGNORE is the ego, but that of REGULATED/PRIORITIZED is the object? They should be the same.
Also, what do you mean by PRIORITIZED in this enum.